brainstormforce/astra
# AST-3170 - v4.1.4 Package Reported Bug-Fixes.

### Description

- Core Button frontend and backend result mismatched
- JIRA : https://brainstormforce.atlassian.net/browse/AST-3176
- When we add WooCommerce sidebar then showing double line on cart page
- JIRA : https://brainstormforce.atlassian.net/browse/AST-3177

### Screenshots

- Core Button frontend and backend result mismatched
- https://d.pr/i/UbhCgE
- When we add WooCommerce sidebar then showing double line on cart page
- https://d.pr/i/HmxmaS

### Types of changes




- Core Button frontend and backend result mismatched - Updated selector in editor to only apply to button elements.
- When we add WooCommerce sidebar then showing double line on cart page - CSS changes to fix UI

### How has this been tested?

- Core Button frontend and backend result mismatched
1. Add a button in editor and frontend and compare if they are similar.


- When we add WooCommerce sidebar then showing double line on cart page
1. Enable Modern Checkout from Customizer.
2. Test with Left Sidebar.
3. Test with Right Sidebar.
4. Test for RTL cases.
5. There should be no UI conflict.

### Checklist:
